pkg/apis/nmstate/v1alpha1/nodenetworkstate_types_test.go

```diff
@@ -29,6 +29,7 @@ status:
       - name: eth1
         type: ethernet
         state: down
+  lastSuccessfulUpdate: null
 
 `
 		nnsStruct = NodeNetworkState{
```

Comment:
You see we create the timestamp on line 42? We should do the same with the new attribute.